client/src/luigi-client.js

```diff
@@ -443,6 +448,55 @@ const LuigiClient = {
           promises.confirmationModal.rejectFn = reject;
         });
         return promises.confirmationModal.promise;
+      },
+
+      /**
+       * Shows an alert.
+       * @param {Object} settings the settings for the alert
+       * @param {string} settings.text the content of the alert. To add a link to the content,
+       * you have to setup the link in `links` object. The key(s) in the `links` object must be used
+       * in the text to reference the links, wrapped in curly braces with no spaces
+       * @param {('info'|'success'|'warning'|'error')} settings.type sets the type of the alert
+       * @param {Object} settings.links provides links data
+       * @param {string} settings.links.LINK_KEY object containing the data for a concrete link.  
+       * Read above on how to use the key to for the link to be properly rendered in the alert message.
+       * @param {string} settings.links.LINK_KEY.text text which will replace the link identifier in the alert content
+       * @param {string} settings.links.LINK_KEY.url url to navigate when the link is clicked. Currently only internal links
+       * are supported in the form of relative or absolute paths.
+       * @returns {promise} which is resolved when the alert is dismissed.
+       * @example
+       * import LuigiClient from '@kyma-project/luigi-client';
+       * const settings = {
+       *  text: Ut enim ad minim veniam, {goToHome} quis nostrud exercitation ullamco {relativePath} laboris nisi ut aliquip ex ea commodo consequat.
+       *    Duis aute irure dolor {goToOtherProject},
+       *  type: 'info',
+       *  links: {
+       *    goToHome: { text: 'homepage', url: '/overview' },
+       *    goToOtherProject: { text: 'other project', url: '/projects/pr2' },
+       *    relativePath: { text: 'relative hide side nav', url: 'hideSideNav' }
+       *  }
+       * }
+       * LuigiClient
+       *  .uxManager()
+       *  .showAlert(settings)
+       *  .then(() => {
+       *     // Logic to execute when the alert is dismissed
+       * });
+
+       */
+      showAlert: function showAlert(settings) {
```

Comment:
I think we should also add a possibility to hide the alert (for example if someone wants to hide it after the navigation).

Reply:
We will discuss it internally and provide the best solution to let the programmer of the microfrontend close the alert without the user explicitly clicking the dismiss button of the alert.